gridt/src/app/movements/movements-detail/movements-detail.page.spec.ts

```diff
@@ -1,30 +1,38 @@
-import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-import { RouterTestingModule } from '@angular/router/testing';
-import { HttpClientModule } from '@angular/common/http';
+import { CUSTOM_ELEMENTS_SCHEMA } from "@angular/core";
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import { RouterTestingModule } from "@angular/router/testing";
+import { HttpClientModule } from "@angular/common/http";
+<<<<<<< HEAD
+=======
```

Comment:
This would most likely not compile or pass the tests. Please run the tests before committing.

Reply:
That is strange... I did run the tests before my last commit and all of them passed.